src/Integration.Vsix/Analysis/AnalysisStrings.resx

```diff
@@ -163,4 +163,16 @@
   <data name="Settings_UsingDefaultSettings" xml:space="preserve">
     <value>Default settings will be used</value>
   </data>
+  <data name="Notifier_AnalysisCancelled" xml:space="preserve">
+    <value>SonarLint: Cancelled analysis of {0}</value>
+  </data>
+  <data name="Notifier_AnalysisEnded" xml:space="preserve">
+    <value>SonarLint: Finished analyzing {0}</value>
+  </data>
+  <data name="Notifier_AnalysisFailed" xml:space="preserve">
+    <value>SonarLint: Failed to analyze {0}, please see the Output Window for more information.</value>
```

Comment:
I wouldn't expect really to see "please" in a status bar message, and I don't think the use a comma here is correct - it should be a new sentence. Just "See the Output Window for more information." would do.